.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: 🐛 Bug report
+description: Report a bug or unexpected behavior
+title: "Bug title"
+labels: []
+body:
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: Please enter a clear and short description of your bug here.
+      placeholder: A crash occurs if X is called from Y.
+  - type: textarea
```

Comment:
A thought: a set of checkboxes for the affected OSes, and then possibly a separate section for additional details?